frontend/src/app/draft/page.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { LinkButton } from "@ui/LinkButton";
+import useDraft from "@services/useDraft";
+
+const Draft = () => {
+  const { isLoading, isError, bookData } = useDraft();
+
+  let lastBook = bookData[bookData.length - 1];
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (isError) {
+    return <div>Error...</div>;
+  }
+
+  return (
+    <section className="w-full py-10 mt-5">
+      <div className="w-4/6 mx-auto">
+        <div className="relative">
+          <Image
+            src="/assets/draft.png"
+            alt="book-title"
+            width={200}
+            height={3}
+          />
+          <div className="absolute top-1/2 left-20 transform -translate-x-1/2 -translate-y-1/2 bg-opacity-70 p-2 whitespace-nowrap">
+            <h1 className="mt-0 font-normal text-3xl md:text-4xl text-green font-quicksand tracking-normal md:tracking-widest leading-none md:leading-normal lg:leading-loose">
+              Draft{" "}
+            </h1>
+          </div>
+        </div>
+      </div>
+      <div className="bg-flower bg-no-repeat bg-right bg-contain">
+        <div className="w-4/6 my-10 mx-auto">
+          <h2 className="text-xl">
+            <span className="font-semibold">Title :</span> {lastBook.title}{" "}
```

Comment:
add font-family: quicksand when necessary! You can refer to a `text` class in tailwing.config file and add more here if needed!